### ISSUE TITLE
How to set somes system as training set?

### ISSUE THREAD

**Issue by wxy20010110** [deepmodeling/deepmd-kit#264]
It seems that deepmd-kit uses certain sets in a system as test sets. Then, if one uses a large amount of system data, how to set some systems as the training set.

**Comment by njzjz** [comment]
deepmd-kit uses the last set as the test set, if the number of sets is more than 1. You can assign one of your test sets as the last set, and remove others.

**Comment by wxy20010110** [comment]
Thanks. But different systems have different type.raw. So I just divide it by myself. And now I'm wondering how to use dp test to test various systems.
